lib/sensor.js

```diff
@@ -223,6 +231,20 @@ function Sensor(opts) {
         return this.constrained;
       }
     },
+    freq: {
+      get: function() {
+        return priv.get(this).freq;
```

Comment:
With the change above, you'll now have access to to `state` from within here, so this can be: `return state.freq;`